### PR TITLE
Add option to use Storybook Server instead of requiring a Static Build

### PR DESCRIPTION
**Pull request by eoinobrien**
Hi all,

I was looking at integrating this into a project and as part of our CI we already have a Storybook Server running. So instead of requiring a static build I figured finding a way to use the server would be nice.

I've added an integration test and validated locally.

Any comments would be appreciated.

Thanks 😄

### CODE REVIEW

**Comment by eoinobrien** [comment]
Looks like the build is failing. It succeeds locally, but I am on a Windows device. I'll take a look at it tomorrow.

File: README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ Option|Default|Type|Description
 -|-|-|-
 `--browser`|`chromium`|chromium, firefox, or webkit|Which browser to run the tests in
 `--build-dir`|`storybook-static`|string|Storybook static build directory
+`--storybook-address`||string|Storybook server address to test against instead of using a static build directory. If set, `--build-dir` will be ignored. e.g. --storybook-address `http://localhost:6006`
```
Comment (eoinobrien) [inline README.md:72]: I'm not fully sure I like the name of this flag. I don't hate it, but I think it could be clearer as to what it does. Any thoughts?

**Comment by eoinobrien** [comment]
@ahuth I went straight to a PR instead of creating an issue first.

This is an issue I noticed while integrating the lib into a project in work. During our CI builds we have a storybook server running to run some tests, so it felt like a bunch of overhead to have a static build of storybook too. This seemed like a simple solution to solve. So here we are.
Let me know if there are any questions, concerns or thoughts at all.

**Review by ahuth — APPROVED** [review]
Oh, interesting. So this will connect to a running Storybook server to get the list of stories. I never thought of that use case, but makes sense.

LGTM, and thanks for contributing this!

**Comment by ahuth** [comment]
@eoinobrien this was published in v3.0.0.

Note that there is also a breaking change in that version from #24. The story parameter `disabled` was renamed to `skip`.

**Pull request merged**